CairoMakie/src/screen.jl

```diff
@@ -68,6 +68,23 @@ function surface_from_output_type(type::RenderType, io, w, h)
     end
 end
 
+@enum PDFVersion PDFv14 PDFv15 PDFv16 PDFv17
+function pdfversion(version::AbstractString)
+    version == "1.4" && return PDFv14
+    version == "1.5" && return PDFv15
+    version == "1.6" && return PDFv16
+    version == "1.7" && return PDFv17
+    throw(ArgumentError("PDF version must be one of '1.4', '1.5', '1.6', '1.7' (received '$version')"))
+end
+
+function cairo_pdf_version(version::PDFVersion)
+    version === PDFv14 && return 0
```

Comment:
this is not necessary, enums already go from 0 to N:

```julia
julia> @enum ENUM A B C

julia> A
A::ENUM = 0

julia> Int(A)
0
```

Reply:
done